cinema/serializers.py

```diff
@@ -30,10 +30,26 @@ class Meta:
         fields = ("id", "name", "rows", "seats_in_row", "capacity")
 
 
+class MovieImageSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = Movie
+        fields = ("id", "image")
+
+
 class MovieSerializer(serializers.ModelSerializer):
     class Meta:
         model = Movie
-        fields = ("id", "title", "description", "duration", "genres", "actors")
+        fields = (
+            "id",
+            "title",
+            "description",
+            "duration",
+            "genres",
+            "actors",
+            "image"
+        )
+        read_only_fields = ("image",)
```

Comment:
Image url should be shown on MovieListSerializer and MovieDetailSerializer

Reply:
read_only_fields = ("image",) This is unnecessary here